bindata/assets/jobset-controller-generated/cert-manager.io_v1_certificate_jobset-metrics-cert.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  labels:
+    app.kubernetes.io/component: certificate
+    app.kubernetes.io/created-by: jobset
+    app.kubernetes.io/instance: metrics-cert
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: certificate
+    app.kubernetes.io/part-of: jobset
+  name: jobset-metrics-cert
+  namespace: openshift-jobset-operator
+spec:
+  commonName: $(METRICS_SERVICE_NAME).$(METRICS_SERVICE_NAMESPACE).svc
```

Comment:
You might need to reduce the length of the `commonName` here otherwise you will end up hitting this issue:
```sh
denied the request: spec.commonName: Too long: must have at most 64 bytes

```

Reply:
updated both in the operator and operand https://github.com/kubernetes-sigs/jobset/pull/863/commits/5a235922c5bcd346bcf16a1b7e313e5b01e7d608